### PR TITLE
Use originoffset

### PR DESCRIPTION
**Pull request by brassev**
This switches to using origin offset instead of a user position.  Hopefully, once originOffset is implemented in Chrome we should just be able to remove some of this stuff and have it work.

### CODE REVIEW

File: src/scripts/scenes/xr-scene.js [hunk 1]
```diff
@@ -152,24 +151,24 @@ export default class XrScene {
...
+  //
+  //   const translationInView = new Matrix4();
+  //   translationInView.makeTranslation(
+  //     tempPosition.x,
+  //     tempPosition.y,
+  //     tempPosition.z
+  //   );
+  //
+  //   viewMatrix.premultiply(translationInView);
+  // }
```
Comment (darktears) [inline src/scripts/scenes/xr-scene.js]: Let's just get rid of that code rather than letting it commented. One can git log to find it back

Comment (brassev) [reply src/scripts/scenes/xr-scene.js]: Removed in updated history.

**Comment by brassev** [comment]
Well, Chrome says they've implemented originOffset: https://chromium.googlesource.com/chromium/src/+/b4a24d48072fd430e45f6912aff75a973235ea87
I'll check how our project works on tomorrow's Canary.

**Comment by darktears** [comment]
Nice!

File: src/scripts/scenes/xr-scene.js [hunk 2]
```diff
@@ -67,11 +65,6 @@ export default class XrScene {
     this._animationCallback();
   }
 
-  _restartAnimation = () => {
```
Comment (jonesjonathan) [inline src/scripts/scenes/xr-scene.js]: This was necessary to keep the animation loops from continuing when an XR session was started.

**Pull request closed without merging**